fix_laser_durations.py

```diff
@@ -0,0 +1,44 @@
+#How to change value in hf5 File due to Laser sampling error (high sampling rate)
+
+import tables
+import numpy as np
+import easygui
+import os
+
+# Ask for the directory where the hdf5 file sits, and change to that directory
+dir_name = easygui.diropenbox()
+os.chdir(dir_name)
+
+# Get the names of all files in the current directory, and find the hdf5 (.h5) file
+file_list = os.listdir('./')
+hdf5_name = ''
+for files in file_list:
+	if files[-2:] == 'h5':
+		hdf5_name = files
+
+# Open up the hdf5 file
+hf5 = tables.open_file(hdf5_name, 'r+')
+
+trains_dig_in = hf5.list_nodes('/spike_trains')
+
+# Ask the user for the correct laser duration and convert to integers
```

Comment:
I see that you are trying to change durations here, but you need to change the onset lags as well because they can have similar sampling errors as the durations. We need to have code to deal with the onset lags too.